Preparing for new plans (#1051)
dnsimple/dnsimple-support: content/articles/dnsimple-services.markdown

## DNS hosting

> We bill DNS hosting as on top of your subscription base fee. Pricing is available on our [pricing page](https://dnsimple.com/pricing).

Reviewer: ```suggestion
Each DNS zone is billed separately from any base subscription fee. Pricing is available on our [pricing page](https://dnsimple.com/pricing).
```
